Sources/Controller/ImagePickerController+Closure.swift

```diff
@@ -45,9 +45,6 @@ import Photos
             imagePicker.onCancel = cancel
             imagePicker.onFinish = finish
 
-            // And since we are using the blocks api. Set ourselfs as delegate
-            imagePicker.imagePickerDelegate = imagePicker
```

Comment:
And if you present it without using the closure API you don't need to remove this.